overrides/config/ftbquests/quests/chapters/quests.snbt

```diff
@@ -12399,7 +12388,7 @@
 					type: "item"
 				}
 				{
-					id: "7E4F26D61480AA55"
+					id: "0811D144BE70E7BB"
```

Comment:
whever quest ids change just double check that any other quest that is supposed to reference it or is dependent is still linked properly! should be fine but just double check